docs/developing.rst

```diff
@@ -0,0 +1,185 @@
+edX Proctoring Developer Guide
+==============================
+
+.. contents::
+
+
+How do I use proctoring on devstack?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Create a test course
+    * Follow the steps here: Including Proctored Exams in Your Course
+* Read the `learner guide for using proctoring <http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/completing_assignments/SFD_proctored_exams.html>`_
+* Start out by trying a practice proctored exam to understand the process
+* The Instructor Dashboard has a "Special Exams" tab for administering proctoring
+    * can add allowances per user, e.g. additional time for an exam
+    * can also reset exam attempts for individual users
+* More debugging can be done in Django admin
+    * "Edx_Proctoring" section
+    * "Proctored exam software secure review" has log of responses from SoftwareSecure
+
+How do I develop on edx-proctoring?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These are the steps to install edx-proctoring into your pre-existing devstack image:
+
+Clone edx-proctoring into the src directory next to edx-platform folder in your host filesystem::
+
+    $ cd src
+    $ git clone https://github.com/edx/edx-proctoring
+    Clone the mock proctoring server repo into your workspace:
+
+    $ cd src
+    $ git clone git@github.com:davestgermain/mockprock.git
+    Install the proctoring package and mockprock into edx-platform in the container, for both LMS and Studio:
+
+Edit or create the ``edx-platform/requirements/private.txt`` file::
+
+    # edx-platform/requirements/private.txt
+    # This file will be used in the docker image, so use file paths that work there.
+    -e /edx/src/edx-proctoring
+    -e /edx/src/mockprock
+
+Then::
+
+    $ make lms-shell
+    root@a7ff4f3f3f6b:/edx/app/edxapp/edx-platform# paver install_prereqs;exit
+    $ make studio-shell
+    root@a7ff4f3f3f6b:/edx/app/edxapp/edx-platform# paver install_prereqs;exit
+
+
+You can also develop edx-proctoring outside of devstack with a virtual environment::
+
+    cd edx-proctoring
+    mkvirtualenv edx-proctoring
+    make upgrade requirements
+    make test-python
+
+In edx-platform/lms/envs/private.py and edx-platform/cms/envs/private.py:
```

Comment:
Can we move the rest of this section (i.e. lines 57-91) to its own section? I think a quick blurb motivating what mockprock is and why it's helpful could be useful.

Reply:
done